conda_smithy/templates/run_docker_build.sh.tmpl

```diff
@@ -60,7 +60,7 @@ fi
 
 export UPLOAD_PACKAGES="${UPLOAD_PACKAGES:-True}"
 {{ docker.executable }} run ${DOCKER_RUN_ARGS} \
-           -v "${RECIPE_ROOT}":/home/conda/recipe_root:ro,z \
+           -v "${RECIPE_ROOT}":/home/conda/recipe_root:rw,z \
```

Comment:
If you are changing this, can you get rid of `RECIPE_ROOT` as well? I don't see any reason to have two mounts here if the same permission is used.

Reply:
This variable is used a lot in the underlying code so changing it will take some work. Let's make an issue and come back.